website/docs/docs/collaborate/govern/project-dependencies.md

```diff
@@ -4,16 +4,16 @@ id: project-dependencies
 sidebar_label: "Project dependencies"
 description: "Reference public models across dbt projects"
 pagination_next: null
+keyword: dbt mesh, project dependencies, ref, cross project ref, project dependencies
 ---
 
 :::info Available in Public Preview for dbt Cloud Enterprise accounts
 
 Project dependencies and cross-project `ref` are features available in [dbt Cloud Enterprise](https://www.getdbt.com/pricing), currently in [Public Preview](/docs/dbt-versions/product-lifecycles#dbt-cloud). 
 
-Enterprise users can use these features by designating a [public model](/docs/collaborate/govern/model-access) and adding a [cross-project ref](#how-to-use-ref).
+Enterprise users can use these features by designating a [public model](/docs/collaborate/govern/model-access) and adding a [cross-project ref](#how-to-write-cross-project-ref).
```

Comment:
```suggestion
[Enterprise users](https://www.getdbt.com/pricing) can use these features by designating a [public model](/docs/collaborate/govern/model-access) and adding a [cross-project ref](#how-to-write-cross-project-ref).
```

Reply:
hey @jtcohen6 so we spoke about this as a docs team and one of the things we all agreed that the main thing is keeping the reader closer to the content (2nd person) so they can check their plan or account status on their own. We align with rephrasing or using  “Enterprise plan or accounts” or “People on enterprise plans can xyz..” to keep users closer to the docs. 

So i made a small tweak to your suggestion, let me know your thoughts. cc @runleonarun @nghi-ly @matthewshaver

Reply:
I will also add some guidance to our style guide